templates/taxi/car_detail.html

```diff
@@ -11,12 +11,23 @@ <h1>
       Update
     </a>
   </h1>
+  <h2>Drivers</h2>
+  <form method="post" action="{% url 'taxi:car-add-or-delete-driver' car.id %}">
```

Comment:
Ensure that the form's action URL correctly matches the URL pattern defined in `urls.py`. The URL name 'taxi:car-add-or-delete-driver' should be consistent with the name used in the URL patterns.